lnd.go

```diff
@@ -675,10 +675,28 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 
 	// With all the relevant chains initialized, we can finally start the
 	// server itself.
-	if err := server.Start(); err != nil {
+	errChan := make(chan error)
+	go server.Start(errChan)
```

Comment:
the alternative would be, seems easier to follow,
```go
	go func() {
                errChan <- server.Start()
        }()
```

Reply:
yes looks better, changed.